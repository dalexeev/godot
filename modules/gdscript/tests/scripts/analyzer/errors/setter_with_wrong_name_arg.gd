var a: int: set = set_named

func test():
	pass

func set_named(name: int, value: int) -> void:
	prints("set_named", name, value)
