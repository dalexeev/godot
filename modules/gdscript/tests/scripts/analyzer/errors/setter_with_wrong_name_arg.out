GDTEST_ANALYZER_ERROR
Function with name argument of type "int" cannot be used as setter for a property.
